spec/system/schools/school_targets_spec.rb

```diff
@@ -201,6 +204,49 @@
 
         it { expect(page).not_to have_link("View results") }
 
+        context 'and there are progress reports for both targets' do
+          let!(:expired_electricity_progress) { build(:fuel_progress, fuel_type: :electricity, progress: 0.99, target: 20, usage: 15) }
+          let!(:expired_gas_progress)         { build(:fuel_progress, fuel_type: :gas, progress: 0.59, target: 19, usage: 17) }
+          let!(:expired_last_generated)       { Date.yesterday }
+
+          let(:expired_target) do
+            create(:school_target, school: test_school,
+              start_date: Date.yesterday.prev_year, target_date: Date.yesterday,
+              electricity: 2, gas: 4,
+              electricity_progress: expired_electricity_progress, gas_progress: expired_gas_progress,
+              report_last_generated: expired_last_generated)
+          end
+
+          let!(:electricity_progress) { build(:fuel_progress, fuel_type: :electricity, progress: 0.90, target: 15, usage: 15) }
+          let!(:gas_progress)         { build(:fuel_progress, fuel_type: :gas, progress: 0.50, target: 7, usage: 17) }
+          let!(:last_generated)       { Time.zone.today }
+
+          let!(:target) do
+              create(:school_target, school: test_school,
+                electricity: 10, gas: 10, storage_heaters: nil,
+                electricity_progress: electricity_progress, gas_progress: gas_progress,
+                report_last_generated: last_generated)
+          end
+
+          it "links to expired progress report" do
+            #Extra check for debugging flickering test
+            expect(Schools::Configuration.count).to be 1
+            expect(School.first.has_electricity?).to be true
+            expect(page).to have_link("View monthly report", href: electricity_school_school_target_progress_index_path(test_school, expired_target))
+          end
+
+          it 'shows progress summary' do
+            within('#electricity-row') do
+              expect(page).to have_content("-2.0%") #target
```

Comment:
Could be helpful to reference expired_electricity_progress.progress and expired_target rather than strings?